dev/tests/acceptance/tests/_suite/msi-suite.xml

```diff
@@ -6,23 +6,107 @@
   */
 -->
 
-<suites xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Suite/etc/suiteSchema.xsd">
+<suites xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Suite/etc/suiteSchema.xsd">
```

Comment:
Please update `app/code/Magento/InventoryAdminUi/Test/Mftf/Suite/msi-suite.xml` instead of this suite.